modules/visualizing-external-entities-known-limitations.adoc

```diff
@@ -9,5 +9,5 @@
 The following are some known limitations of the Visualizing external entities feature:
 
 * When you enable external IP collection for a cluster, Collector in those clusters report more information to Sensor and to Central. This might create scalability issues if the workload in the cluster communicates with a large number of distinct external peers. It is recommended that you do not enable this feature on clusters with communication patterns involving more than 10,000 distinct external entities.
-* The display of IP information on the *Network Graph* page is experimental and uses a graphical item for each IP entity. Enabling this level of detail when the number of peers is a few hundred or more might result in an unreadable user interface and  more than 2,000 peers may result in a potentially unresponsive user interface.
 * You cannot see external IP addresses if they are part of CIDR blocks.
+* When you enable external IP collection, external IP addresses may appear in a deployment's network baseline.
```

Comment:
```suggestion
* When you enable external IP collection, external IP addresses might appear in a deployment's network baseline.
```
We don't use "may" in technical documentation - per [IBM Style guide](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#may) (you can read with your redhat email as the ID, if you really want to....) :)

Reply:
Done